django/contrib/auth/views.py

```diff
@@ -185,7 +185,7 @@ def logout_then_login(request, login_url=None, extra_context=None):
     if not login_url:
         login_url = settings.LOGIN_URL
     login_url = resolve_url(login_url)
-    return logout(request, login_url, extra_context=extra_context)
+    return LogoutView.as_view(next_page=login_url, extra_context=extra_context)(request, login_url, extra_context)
```

Comment:
@timgraham I am not sure if we really need `extra_content` at all. context makes sense when we are rendering some template but with `logout_then_login` we just have `HttpRedirectResponse` with no context data. Details here https://github.com/andrewnester/django/blob/47f8f72b8a9980d915131b9971209892e18b5559/django/contrib/auth/views.py#L136